frontend/src/components/navbar.js

```diff
@@ -1,57 +1,70 @@
-import React, { useState } from 'react';
-import { Navbar, Form, Button, NavbarToggler, NavbarBrand, Nav, NavItem, NavLink, InputGroup, Input } from 'reactstrap';
-import { BrowserRouter as Router, Switch, Route} from 'react-router-dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'antd/dist/antd.css';
+import { Menu } from 'antd';
+import { QuestionOutlined, GithubOutlined, HomeOutlined, UserOutlined, LoginOutlined, PlusOutlined, AppstoreAddOutlined } from '@ant-design/icons';
 
+import { BrowserRouter as Router, Switch, Route} from 'react-router-dom';
+import '../styles/navbar.css' 
 import homePage from './home'
-
+import concertPlaylist from './concertPlaylist'
 import playlistPage from './playlist'
 import userPage from './user.js'
 import aboutUsPage from './aboutUs'
 
+const { SubMenu } = Menu;
 
-const Example = (props) => {
-    const [isOpen, setIsOpen] = useState(false);
-
-    const toggle = () => setIsOpen(!isOpen);
-
-  return (  
-      <Router>
-        <div>
-          <Navbar color='light' light expand='md'>
-          <NavbarToggler onClick={toggle} className='mr-2' />
-            <NavbarBrand href="/">PLAYLISTAPP</NavbarBrand>
-              <Nav className='mr-auto' navbar tabs>
-              <NavItem>
-                  <NavLink href="/playlist">PLAYLIST</NavLink>
-                </NavItem>
-                <NavItem>
-                  <NavLink href="/user">USER</NavLink>
-                </NavItem>
-                <NavItem>
-                  <NavLink href="/aboutus">ABOUT</NavLink>
-                </NavItem>
-                <NavItem>
-                  <NavLink href="http://localhost:8888">LOGIN</NavLink>
-                </NavItem>
-              </Nav>
-              <Form inline>
-                <InputGroup>
-                    <Input placeholder = 'keyword'></Input>
-                </InputGroup>
-                <Button variant='outline-success'>Search</Button>
-              </Form>        
-            </Navbar>
-          </div>
-          <Switch>
+class NavBar extends React.Component {
+  state = {
```

Comment:
Can we put this inside of a [constructor](handleClick) method since we are setting state here.

Reply:
Done.